src/contrib/highlight.ts

```diff
@@ -6,7 +6,7 @@
  * - Modified by Brian Reavis <brian@thirdroute.com> 2012-8-27 (cleanup)
  */
 
-import {replaceNode} from '../vanilla';
+import {replaceNode} from '../vanilla.ts';
```

Comment:
These extensions make the ESM build work properly. `tsc` will transform these appropriately during build.